tests/e2e_tests/multistep/test_axon.py

```diff
@@ -96,13 +96,9 @@ async def test_axon(local_chain):
     # record logs of process
     # Create tasks to read stdout and stderr concurrently
     # ignore, don't await coroutine, just write logs to file
-    await asyncio.create_task(
-        write_output_log_to_file("axon_stdout", axon_process.stdout)
-    )
+    asyncio.create_task(write_output_log_to_file("axon_stdout", axon_process.stdout))
     # ignore, dont await coroutine, just write logs to file
-    await asyncio.create_task(
-        write_output_log_to_file("axon_stderr", axon_process.stderr)
-    )
+    asyncio.create_task(write_output_log_to_file("axon_stderr", axon_process.stderr))
```

Comment:
These need to be awaited, or else they won't write the logs to the file.

Reply:
According to Daniel, he uses this for debugging, so he asked to ignore it.